trio/_abc.py

```diff
@@ -486,10 +486,10 @@ class Listener(AsyncResource):
         """Wait until an incoming connection arrives, and then return it.
 
         Returns:
-          AsyncResource: an object representing the incoming connection. In
-              practice this is almost always some variety of :class:`Stream`,
-              though in principle you could also use this interface with, say,
-              SOCK_SEQPACKET sockets or similar.
+          AsyncResource: An object representing the incoming connection. In
+          practice this is almost always some variety of :class:`Stream`,
+          though in principle you could also use this interface with, say,
+          SOCK_SEQPACKET sockets or similar.
```

Comment:
Does this still render correctly with the changed indentation? I can never predict what Sphinx will do with these kinds of things...

Reply:
Yes, i built the docs locally to make sure it is fixed. I could upload a pic of the fixed layout later.
This is the current layout to be fixed:
![current_layout](https://user-images.githubusercontent.com/5450806/40345907-ae1152bc-5da3-11e8-92b8-1de9f1418f45.png)
(notice the first line of "Returns" section is in its own box, and the rest is idented)

Reply:
Haha, whoops, that does look terrible. Thanks :-)